datasets/librispeech.py

```diff
@@ -0,0 +1,68 @@
+import json
+import pathlib
+import numpy as np
+import librosa
+
+BASEDIR = pathlib.Path("/home/woze/projects/tinygrad/datasets/librispeech")
```

Comment:
Use __file__ to not code directories specific to your machine.

Also, I'm fine with not having an auto download script for the dataset, but include your top GitHub comment as a comment in here so it works standalone.